lti_emailer/settings/production.py

```diff
@@ -15,3 +15,11 @@
 
 # Allow the mailing_list app to allow any user to be added to listserv
 IGNORE_WHITELIST = True
+
+# TODO - supply huey.crontab() arguments to determine how often the periodic
+#        mailing list membership sync happens.  defaults to every 1 hour.
+LISTSERV_PERIODIC_SYNC_CRONTAB = {}
+
+# TODO - point huey to the production redis/elasticache server.  we want to
+#        override HUEY['consumer_options'] here.
```

Comment:
Is that where we're putting all the env-specific settings now?  Sure, this was just a placeholder for when we talked to devops about deploying.